### ISSUE TITLE
alternative content for member detail graph and map

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#638]
from [john's review](https://docs.google.com/spreadsheets/d/1N-oBy9Mp91TEdXEi6rjCMNsbJNacxMh6CEGTHpCRfws/edit?usp=sharing):
> There does not appear to be an accessible version of the charts and map on the member details page.

we definitely at least have a description attribute linked to the graph, so _something_ should be getting read. need to check what the leaflet map sounds like.

**Comment by rlskoeser** [comment]
@thatbudakguy I was struggling to get voiceover to work since it's been so long since I used it, but I did discover that I was setting a useless `aria-described-by` attribute on the SVG instead of `aria-describedby`. 🤦‍♀️ 

Does it matter that the SVG is added by code? I never successfully heard the description after I changed the attribute, but I'm inclined to attribute that to my difficulties using VO.

Not sure what to do about the map.
